package.json

```diff
@@ -56,6 +56,7 @@
     "typescript": "^4.3.5"
   },
   "dependencies": {
+    "@byndyusoft/nest-dynamic-module": "https://github.com/Byndyusoft/nest-dynamic-module/releases/download/init-snapshot/nest-dynamic-module-0.0.0-development-feature-init-1.tgz",
```

Comment:
It's ok?

Reply:
I fix it when `nest-dynamic-module` will be released in npm.